ward/collect.py

```diff
@@ -158,3 +161,28 @@ def search_generally(
 
         if matches_query and matches_tags:
             yield test
+
+
+def filter_fixtures(
```

Comment:
I missed this on my earlier reviews but spotted it due to the new test coverage analysis added recently: I'd appreciate if you would add some tests for this function.

After those are in I think we're good to go!

Reply:
Done, though I'm not totally happy with them. At some point I'd like to do a pass and build some higher-level abstractions for testing fixtures, like there are for testing tests.